src/main/java/com/github/tomakehurst/wiremock/standalone/CommandLineOptions.java

```diff
@@ -134,6 +135,7 @@ public CommandLineOptions(String... args) {
         optionParser.accepts(ASYNCHRONOUS_RESPONSE_ENABLED, "Enable asynchronous response").withRequiredArg().defaultsTo("false");
         optionParser.accepts(ASYNCHRONOUS_RESPONSE_THREADS, "Number of asynchronous response threads").withRequiredArg().defaultsTo("10");
         optionParser.accepts(USE_CHUNKED_ENCODING, "Whether to use Transfer-Encoding: chunked in responses. Can be set to always, never or body_file.").withRequiredArg().defaultsTo("always");
+        optionParser.accepts(NOTIFIER, "Use a specific notifier. Can be either ConsoleNotifier, Slf4jNotifier or a full identifier").withRequiredArg().defaultsTo("ConsoleNotifier");
```

Comment:
> Slf4jNotifier or a full identifier

Fully qualified name of the class which implements Notifier interface. This will be clear, I gues